src/src/components/LabelAndValueModule.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './switch.css';
+
+
+
+const LabelAndValueModule = (props: any) => {

```

Comment:
If these modules are only to be consumed by the toolbar, I would do a module for the Toolbar and set its child components under it:
```
toolbarmodule
    ToolbarModule.tsx
    modules
      LabelAndValueModule.tsx
      BatteryLevelIndicatorModule.tsx
```
Thoughts?

Reply:
The `LabelAndValueModule` is a generic module that can be used in any other module.